organization/documentation.md

```diff
@@ -4,6 +4,18 @@ documentation should be easy to edit (wiki-style), versioned (git) and able to o
 
 a near wiki-like experience can be achieved with sphinx doc being built and pushed immediately to prod as a CI actions from github when a commit is pushed. Repo has no named branches other than develop, and represent a continuous line in time.
 
+```{note}
+About maintaining doc pages hierarchy, tags, categories
+
+* Several categories of items (besides what and how) should be added and the hierarchy of categories is not known in advance.
+
+* With time the hierarchy or tags tends to degenerate. When adding new items to the hierarchy I tend to have forgoten the location of previous semantically-related items or the tags I used for then, to the items end up being spread around. Out of my personal experience this happened in a team (company wiki) and in my personal bookmarks repository. 
+
+* Also categories and tags tend to evolve in time. I first filed related context under `neural networks`, the `neural networks->transformer` then `neural networks->transformer->llm`, then `neural networks->transformer->llm->rag`
```

Comment:
agreed, which is why we should look for documentation tools that allow us to change/edit/correct information quickly and easily. We want as little friction as possible when writing documentation